RedHatInsights/compliance-frontend
# feat(RulesTable): RHICOMPL-3418 - Add editing of a value to rules' details component in RulesTable

This adds value editing for rules.

How to test:

1) Create a policy via the wizard
2) On the rules step look for a rule that has values for example:
    * Set SSH Daemon LogLevel to VERBOSE
    * Modify the System Login Banner (Should show value, but not allow editing)
3) Save the policy and navigate to it details page to the "Rules" tab
4) Search for the same rules you have set a value for and verify it shows the value(s) set
5) Test resetting the value to it's default via the reload icon
6) Edit the value again and verify it is save correctly
7) Edit another value and verify that all values are saved and persist and none are wiped
6) Open the "Edit Policy" modal via the "Edit Rules" button
8) Verify again rules are shown correctly and it is possible to edit them
9) Save and verify the policy details also reflects the changes

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
